CHANGELOG.md

```diff
@@ -15,6 +15,18 @@ For reference, the possible headings are:
 * **External Contributors** to list contributors outside of BigchainDB GmbH.
 * **Notes**
 
+
+## [1.0.1] - 2017-07-12
+Tag name: v1.0.1
+
+### Fixed
+* Mistakenly changed MongoDB version via PR [#1641](https://github.com/bigchaindb/bigchaindb/pull/1641).
+* Quickstart guide via PR [#1648](https://github.com/bigchaindb/bigchaindb/pull/1648).
```

Comment:
I would bundle the first two fixes into (Fixed) "Various issues in the Quickstart page". Pull requests #1641 and #1648